sample.ini

```diff
@@ -1,29 +1,37 @@
 [General]
-
-hostname = 0.0.0.0
 
-portNumber = 8000
+hostname = 0.0.0.0
 
-enableTracing = 0
+portNumber = 8000
+
+enableTracing = 0
 
 ; If any symbols of interest aren't available locally (e.g. Windows DLLs), uncomment line below
-
+
 ; remoteSymbolServer = http://symbolapi.mozilla.org:80/
 
-
-
-; These options are unlikely to need editing by users
 
-maxCacheEntries = 2000000
 
-mruSymbolStateFile = /tmp/snappy-mru-symbols.json
+; These options are unlikely to need editing by users
+
+maxCacheEntries = 2000000
+
+mruSymbolStateFile = /var/snappy-mru-symbols.json
```

Comment:
hrm, I fear this might bring inconsistency for configuration across files. We still have a mix of CRLF/LF files in the repository. sample.ini was more problematic because its line endings had an extra LF. One way to address this is adding a [.gitattributes](https://www.kernel.org/pub/software/scm/git/docs/gitattributes.html) file with the following:

```
* text=auto
```

This will enforce files go into the repository with LFs. Windows developers can set their git configuration [core.autocrlf](https://help.github.com/articles/dealing-with-line-endings/) to:

```
git config --global core.autocrlf true
```

Then they will get CRLF for checked out files, but commits will store LF in the repo.